libs/components/molecules/src/lib/interactive-svg/interactive-svg.component.ts

```diff
@@ -74,8 +74,10 @@ export class InteractiveSvgComponent implements OnDestroy {
   /** SVG url */
   @Input() url?: string;
 
+  @Input() mapping: Record<string, string>[] = [];
```

Comment:
You won't be able to use `MapEntry` in this file. Instead create and export an interface from this file. I.e.
```ts
export interface NodeMapEntry {
  node_name: string;
  label: string
}
```
then add a generic parameter to the class i.e.
```ts
class InteractiveSvgComponent<T extends NodeMapEntry> { ... }
```
and replace uses of `Record<...>` with `T`